metadata/src/main/java/org/apache/kafka/controller/QuorumController.java

```diff
@@ -1010,6 +1004,11 @@ private void maybeCompleteAuthorizerInitialLoad() {
         }
     }
 
+    // Visible for testing
+    MetadataVersion currentMetadataVersion() {
+        return featureControl.metadataVersion();
+    }
```

Comment:
Which test needs this? It doesn't look like this is used in this PR.

Reply:
Oops, that comment was copied over from another PR #12177  :)